packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -139,6 +140,13 @@ describe('NetworkController', () => {
       uuidCounter += 1;
       return uuid;
     });
+
+    jest
+      .spyOn(
+        LastUpdatedAtNetworkConfigurationModule,
+        'updateNetworkConfigurationLastUpdatedAt',
+      )
+      .mockImplementation((n) => n);
```

Comment:
Since the `lastUpdated` field is updated on every network add and network update, it would mean we need to update almost every add/update test 😅 

And you've pinned the nail on the head, we extracted and mocked this function just so we don't add the `updatedAt` field, so the existing tests don't need changing.

... Yeah this is me probably being lazy. If the intention is we want to have a true 1:1 of the network state, then I'll just `jest.fakeTimers` and update all the tests.

Reply:
To be fair @Prithpal-Sooriya I've had to update all the `AccountsController` UTs to support the new `nameLastUpdatedAt` field 😛 . I know it's cumbersome but at least it's there to stay, I'd be in favor of doing that

Reply:
> As this test is not truthful anymore. It does call underlying logic which does end up updating/adding the `updatedAt` property. Maybe treat it as a small lie?

Tests ought to describe the current behavior of the system, so it would be better if we didn't lie :)

It seems that we have two options: either we change the name of the test to say that only `updatedAt` was changed. Or maybe we add some logic so that we don't update `updatedAt` if there are no changes to the network configuration?

Reply:
I'll explore both, thanks for the feedback!